app/main.py

```diff
@@ -1 +1,5 @@
-# write your code here
+def copy_file(command: str):
+    lst = command.split(" ")
+    if lst[0] == "cp" and lst[1] != lst[2]:
+        with open(f"{lst[1]}", "r") as file_in, open(f"{lst[2]}", "r") as file_out:
```

Comment:
Does this code realy working? Because I see you opened file_out for reading also.